### PR TITLE
[WDA-1474] New AgentD endpoints

### PR DESCRIPTION
**Pull request by bogue**

### CODE REVIEW

**Review by manuquentin — DISMISSED** [review]

**Comment by manuquentin** [comment]
I see that there is no doc about agentd. Can you please add it ? (And add it in the doc menu) ?

File: src/api/agentd.js [hunk 1]
```diff
@@ -45,4 +45,36 @@ export default (client: ApiRequester, baseUrl: string) => ({
...
     ),
+
+  staticLogin: () =>
+    client.post(
+      `${baseUrl}/users/me/agents/login`,
+      null,
+      null,
+      ApiRequester.successResponseParser,
+    ),
+
```
Comment (fblackburn1) [inline src/api/agentd.js:56]: Any reason to duplicate the `loginWithLineId` method?

File: src/api/agentd.js [hunk 2]
```diff
@@ -45,4 +45,36 @@ export default (client: ApiRequester, baseUrl: string) => ({
...
+  staticLogout: () =>
+    client.post(
+      `${baseUrl}/users/me/agent/logout`,
```
Comment (fblackburn1) [inline src/api/agentd.js:59]:
```suggestion
      `${baseUrl}/users/me/agent/logoff`,
```

**Review by fblackburn1 — DISMISSED** [review]

File: src/api/agentd.js (hunk 1 above)
Comment (bogue) [reply src/api/agentd.js:56]: lol -- never realized `loginWithLineId` was already using the `/user/me/agents` route, i'd just blindly inserted placeholder code while waiting for the specs to formalize. Consider it fixed.

**Review by manuquentin — APPROVED** [review]

**Pull request merged**